### DISCUSSION TITLE
Non Admin User access to Frigate NVR

### DISCUSSION THREAD

**Discussion by Sarelseekat** [blakeblackshear/frigate#1766] [General]
Hi there

I have some non-Admin users that I would like to give access to the Frigate NVR in the side panel of Home Assistant. At the moment Non-Admin users can't see Frigate side panel. The Lovelace card works but has some delays in displaying feeds. Many of the Lovelace buttons are problematic and it would be much simpler to just have access to Frigate in the side panel. The lovelace card has its place, but the lag and delays create a bigger need for other users to access the NVR directly.

Can this be done? Is there a better way to stream video to a Lovelace Card?

**Comment by blakeblackshear** [comment]
Just updated the addons to enable this. Also, there are some coming updates to the lovelace card that will a major improvement in streaming directly in home assistant.

**Reply by Sarelseekat** [reply]
Fantastic news - Thanks Blake

**Reply by mathgim** [reply]
Cool! Is there a way to prevent non-admin users to access it? Or at least not touch to the four config items? Config, Storage, etc...

**Comment by lilmansplace** [comment]
I was looking forward to this feature.  I see the side panel link but when I click it: 
![image](https://user-images.githubusercontent.com/2847532/136822918-f2d43659-2a7f-4a9d-93f8-13955c0a6076.png)
It doesn't work.  Did I do something wrong?

**Reply by Apeiv** [reply]
I have the same problem..

**Reply by blakeblackshear** [reply]
Does this only happen for non-admin users?

**Reply by Apeiv** [reply]
Yes, for me only non-admin users

**Reply by blakeblackshear** [reply]
Can you check the supervisor logs and or the browser console logs?

**Reply by Apeiv** [reply]
Sure, below the logs that I found.

Browser chrome console:
![image](https://user-images.githubusercontent.com/88326415/136973088-e1d61e0e-e4b6-4905-8011-e37624f7b3a6.png)

Home assistant Logs:

2021-10-12 16:10:06 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from host-cccccc (xx.x..xxx.x). (Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.71 Safari/537.36)
2021-10-12 16:10:15 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [140387418056304] Error handling message: Unauthorized

**Reply by lilmansplace** [reply]
As mentioned by @Apeiv I get the same console errors.  It only happens to the non-admin users
